WebRoot/static/dhtml/main.js

```diff
@@ -1,160 +1,131 @@
-// ECMAScript 5
-"use strict";
+// Global Variables
+let url = window.location.href;
+let body = document.body.style;
+let icon = document.getElementById("menu-icon");
+let winWidth = window.innerWidth;
 
-function sendEmail()
-{
-	var email = document.getElementById("email").value;
-	var form = new FormData();
-	var xhr = new XMLHttpRequest();
+/**
+ * Functions that run automatically
+ */
 
-	form.append("email", email);
+window.addEventListener("load", onLoad());
 
-	xhr.open("post", "/ajax/email/send-code.php");
-	xhr.send(form);
+function onLoad()
+{
+	replaceMultipleBrokenImgs(); //replace broken image url links
 }
 
-// Navbar and Filter on mobile
-const url = window.location.href;
-const body = document.body;
-const icon = document.getElementById("menu-icon");
-const navbar = document.querySelector(".navbar-box");
-const bkg = document.querySelector(".navbar-background");
-const filter = document.getElementById("filter");
+window.addEventListener("resize", () => 
+{
+	let navbar = document.querySelector(".navbar-box").style;
+	let bkg = document.querySelector(".navbar-background").style;
+	let filter = document.getElementById("filter").style;
```

Comment:
It's simple, to stay compatible with ES5 just replace all "let" and "const" by "var". That's it.

Reply:
hahahahah